Snippets/ASB/ASB_7/CustomOversizedBrokeredMessageHandler.cs

```diff
@@ -0,0 +1,14 @@
+using System.Threading.Tasks;
+using Microsoft.ServiceBus.Messaging;
+using NServiceBus.Transport.AzureServiceBus;
+
+#region asb-configure-oversized-messages-handler
+internal class CustomOversizedBrokeredMessageHandler : IHandleOversizedBrokeredMessages
+{
+    public Task Handle(BrokeredMessage brokeredMessage)
+    {
+        // do something usefull with the brokered message, f.e. store it in blob storage
```

Comment:
i dont think f.e. is a well know replacement for "for example". do u mean e.g. ?